server/flaskr/services/notifications/notification_service.py

```
@@ -0,0 +1,55 @@
+        firebase_admin.initialize_app(cred)
+
+    @classmethod
+    def add_notification(cls, type, orgId, *arg, **kwarg):
+        handler = handlers.get(type)
+        assert handler is not None
+
+        job = handler.create(orgId, *arg, **kwarg)
+        NotificationJobDataHandler.create_notification_job(job)
+
+    @classmethod
+    def send_loop(cls):
```

Ruff (0.15.20)

[warning] Missing return type annotation for classmethod `send_loop`

Add return type annotation: `None`

(ANN206)